CHANGELOG.md

```diff
@@ -33,7 +33,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Contribution POST gives proper error message. [#714](https://github.com/rokwire/rokwire-building-blocks-api/issues/714)
 - Hard coded base url. [#780](https://github.com/rokwire/rokwire-building-blocks-api/issues/780)
 - Multiple bugs/improvements around login. [#782](https://github.com/rokwire/rokwire-building-blocks-api/issues/782)
-
+- Fix login problem with session. [#796](https://github.com/rokwire/rokwire-building-blocks-api/issues/796)
```

Comment:
The CHANGELOG entries need to be moved under the unreleased section. Thanks.

Reply:
moved to unreleased

Reply:
> Please fix the merge conflict too. This looks good. Logout button is not present on Contribute page. I guess it can be addressed in another task.

fixed the conflict